### PR TITLE
XRWH-5 Updated icons and styles for go to top button, prev/next butto…

### PR DESCRIPTION
**Pull request by vasyl-krupa**
…ns, breascrumbs separator, top buttons

### CODE REVIEW

**Review by yuliia-biletska — APPROVED** [review]
approved

File: plugins/rocks.xml.webhelp/xsl/topic.xsl [hunk 1]
```diff
@@ -117,10 +117,11 @@
           <div class="breadcrumb-container max-width">
                 <xsl:call-template name="generate-custom-breadcrumbs"/>
             </div>
-          
+
             <div class="top-nav-buttons-container max-width">
-                <a class="prev-button" href="#">&#8592; PREV</a>
-                <a class="next-button" href="#">NEXT &#8594;</a>
+                <!-- TODO: Replace with working prev/next buttons -->
+                <a class="prev-button" href="#"><pre class="button-prev-nav"/> PREV</a>
```
Comment (ibarinova) [inline plugins/rocks.xml.webhelp/xsl/topic.xsl]: We should use default dita-ot 'getVariable' template here and use translated strings.

Comment (vasyl-krupa) [reply plugins/rocks.xml.webhelp/xsl/topic.xsl]: OK, will do

**Review by ibarinova — APPROVED** [review]
Looks good.

**Pull request merged**